### ISSUE TITLE
Update "Use Contao 4.9 in Symfony application"

### ISSUE THREAD

**Issue by bytehead** [contao/docs#593]
For me as a reminder until https://github.com/contao/contao/pull/2321 is released.

**Comment by bytehead** [comment]
@fritzmg can you assign me this please, thanks!

**Comment by fritzmg** [comment]
@bytehead is this still relevant?

**Comment by bytehead** [comment]
Let me check this week.

**Comment by fritzmg** [comment]
Someone on the Community Forum has troubles installing the `contao/core-bundle` into a Symfony Application. I think there are some changes necessary regarding the 2FA things?

https://community.contao.org/de/showthread.php?80295-Contao-Core-Bundle-Installation-schl%C3%A4gt-fehl

**Comment by bytehead** [comment]
Yep, there is 🙈 I'll answer in the forum.

In `config/packages/scheb_two_factor.yaml`:
```yaml
# Scheb 2FA configuration
scheb_two_factor:
    trusted_device:
        enabled: true
    backup_codes:
        enabled: true
```

**Comment by bytehead** [comment]
Closed in favor of #670.
